.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -296,6 +296,23 @@ Minikube is the only caveat - it's not reachable publicly so don't run a proxy t
       * [ ] Restart the agent after token TTL expires to see if it reuses the same identity.
     * [ ] Force cluster CA rotation
 
+### Kubernetes Pod RBAC
+
+* [ ] Check if Roles prior to `v6` are allowed to access all pods.
```

Comment:
Do we need this in the test plan? Sounds like a test for `v6` role implementation, no something that we want to test before each release.

Reply:
Yeah, I will include it just for Teleport 12 test plan